### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by effron**
/no-platform

Bump version number for github packages release

### CODE REVIEW

File: clockwork.gemspec [hunk 1]
```diff
@@ -1,6 +1,10 @@
+$:.push File.expand_path('lib', __dir__)
+
+require "clockwork/version"
```
[the remark this answers is not in the document]
Comment (effron) [reply clockwork.gemspec]: good call!

**Review by rzane — DISMISSED** [review]
domain lgtm

File: clockwork.gemspec [hunk 2]
```diff
@@ -1,6 +1,8 @@
+require_relative "clockwork/version"
```
Comment (rzane) [inline clockwork.gemspec:1]: Oops, I think this needs to be `lib/clockwork/version`.

**Review by rzane — APPROVED** [review]
domain lgtm

**Pull request merged**